common/src/main/scala/org/apache/celeborn/common/meta/WorkerInfo.scala

```diff
@@ -215,6 +215,7 @@ class WorkerInfo(
         diskInfos.remove(nonExistsMountPoint)
       }
     }
+    diskInfos
```

Comment:
Shall we return a copy of diskInfos to avoid potential concurrent issues? In that way we might wrap a new method, i.e. updateDiskInfos and updateThenGetDiskInfos.

Reply:
> updateThenGetDiskInfos

updated, how about current?